### DISCUSSION TITLE
Welcome to wd-syncthing-1.11.1 Discussions!

### DISCUSSION THREAD

**Discussion by soumyamaity** [soumyamaity/wd-syncthing-1.11.1#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kmanan** [comment]
Does this still work with WD NAS EX2 Ultra? I get an error that 1.11.1 is not compatible
